test/helpers/application_helper_test.rb

```diff
@@ -0,0 +1,17 @@
+require 'test_helper'
+
+class ApplicationHelperTest < ActionView::TestCase
+  test 'warning_svg' do
```

Comment:
This is just testing the method against the constant. It will pass even when the constant is modified. So we can skip this test and the test for star_svg.